README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
```

Comment:
I don't want this to be an index of papers. The data will never be 'up to date' again. This project was originally driven by an issue of getting up to date and comprehensive data sets.

Reply:
True. I'm not aware of a live hosted dataset equivalent.  perhaps we need a category of 'dataset's needed?